app/styles/app.scss

```diff
@@ -36,8 +36,13 @@ body {
 
 .toolbar {
 	height: 60px;
-	border-bottom: 1px solid #e4e2e2;
-	font-size:16px;
+	font-size: 13px;
+  background: #e1563f top center;
+	background-image: url("http://emberjs.com/images/navigation_background.png");
```

Comment:
@rwjblue Definitely. I was just setting this up quickly to get feedback. Will fix this right away.